scripts/completion-tests/completionTests.sh

```diff
@@ -33,8 +33,8 @@ EOF
 _completionTests_verifyCompletion "helm stat" "status"
 _completionTests_verifyCompletion "helm status" "status"
 _completionTests_verifyCompletion "helm lis" "list"
-_completionTests_verifyCompletion "helm r" "registry repo rollback"
-_completionTests_verifyCompletion "helm re" "registry repo"
+_completionTests_verifyCompletion "helm r" "repo rollback"
+_completionTests_verifyCompletion "helm re" "repo"
```

Comment:
The registry command is now hidden as experimental (on dev-v3), so these were failing